contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -4184,12 +4213,15 @@ TdsSendSpatialHelper(FmgrInfo *finfo, Datum value, void *vMetaData, int TdsInstr
     *((int32_t*)buf) = srid;
     itr = buf + 4;
 
-    /* Driver Expects 01 0C as 2 constant Bytes */
-    /* TODO: Will need to verify for Different Geometry Data Types */
-    *itr = 1;
-    itr++;
-    *itr = 12;
-    itr++;
+	/* Driver Expects 01 0C for 2-D Point Type as 2 constant Bytes to identify the Geometry Type */
+	/* TODO: Will need to introduce for Different Geometry Data Types */
+	if(*((uint32_t*)gser->data) == 1)
```

Comment:
Are we checking if it is a POINT? Instead of silently ignoring other shapes, we can throw an error here.

Reply:
Done